Filter out '/usr/include' for swig to avoid host contamination issue.

Upstream-Status: Inappropriate [cross compile specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 5a12981..389d49f 100644
--- a/setup.py
+++ b/setup.py
@@ -153,7 +153,8 @@ class _M2CryptoBuildExt(build_ext.build_ext):
             self.swig_opts.append('-py3')
 
         # swig seems to need the default header file directories
-        self.swig_opts.extend(['-I%s' % i for i in _get_additional_includes()])
+        self.swig_opts.extend(['-I%s' % i for i in _get_additional_includes()
+                                            if i != '/usr/include'])
 
         log.debug('self.include_dirs = %s', self.include_dirs)
         log.debug('self.library_dirs = %s', self.library_dirs)
